scripts/tools/release-cli/replaceInAllFiles.sh

```diff
@@ -24,6 +24,7 @@ SOURCES=(
   "core/src"
   "client/src"
   "client/luigi-client.d.ts"
+  "container/src"
```

Comment:
I think for container the replacement is done separately on `container/prepareNextRelease.js` [here](https://github.com/SAP/luigi/blob/main/container/prepareNextRelease.js). I am thinking this might not be needed 🤔